components/MobileMenu/index.tsx

```diff
@@ -21,8 +21,8 @@ const MobileMenu = ({ links }: MobileMenu) => {
             leaveFrom='transform scale-100 opacity-100 translate-y-0'
             leaveTo='transform scale-95 opacity-0 translate-y-10'
           >
-            <Popover.Panel className='flex flex-col items-end'>
-              <div className='bg-primary flex justify-around flex-col items-center p-3 rounded-lg w-full h-96 font-serif'>
+            <Popover.Panel className='flex flex-col items-end justify-end h-screen'>
```

Comment:
not sure why this needs to be `h-screen`

Reply:
Because I wanted the height of the Meny to be a percentage of the screen(i.e. half)
Which required the div parent of the menu to have a height equal to that of the screen